### ISSUE TITLE
Query doesn't work with extension

### ISSUE THREAD

**Issue by dimarudik** [postgredients/pg_stat_query_plans#3]
Hi Leonid,

The query below doesn't work with extension
It fails with error
ERROR:  failed to find plan for subquery fatal_query

WITH abc AS
(
    SELECT
      (fatal_query.keys).n,
      'abrakadabra'::text AS c_text
    FROM  
    (
      SELECT information_schema._pg_expandarray(ikey) AS keys
      FROM
      (
        SELECT array[1,2] AS ikey
        -- SELECT array[]::text[] -- или так
      ) AS c1
    ) AS fatal_query
)
SELECT * FROM abc WHERE c_text = '"abrakadabra"';

**Comment by leborchuk** [comment]
Looks like bug in explain verbose https://postgrespro.com/list/id/dfe47e78-f9c3-4f59-a345-591f7fd3ceec@app.fastmail.com

I thought pg_stat_query_plans.example_plan_verbose could help here but not, in 745 line I set definitely es_planid->verbose = true; to get normalized execution plan

Hmmm, try to fix it )

**Comment by Kokokokoka** [comment]
>Looks like bug in explain verbose https://postgrespro.com/list/id/dfe47e78-f9c3-4f59-a345-591f7fd3ceec@app.fastmail.com

There is also a new report https://postgrespro.com/list/id/18576-9feac34e132fea9e@postgresql.org#head

**Comment by Kokokokoka** [comment]
REL_16_STABLE branch now has a fix for this query (and every 13+ branch too). Tested it with this extension enabled, this query now works. Fixed with commit: 03f679475d9ae6cafa1ea155e9f77255f245ef79.
```
test-# SELECT * FROM abc WHERE c_text = '"abrakadabra"';
 n | c_text
---+--------
(0 rows)

test=# EXPLAIN (VERBOSE) WITH abc AS                                                                                                                                           (                                                                                                                                                                            SELECT                                                                                                                                                                       (fatal_query.keys).n,                                                                                                                                                        'abrakadabra'::text AS c_text                                                                                                                                                FROM                                                                                                                                                                         (                                                                                                                                                                            SELECT information_schema._pg_expandarray(ikey) AS keys                                                                                                                      FROM                                                                                                                                                                         (                                                                                                                                                                            SELECT array[1,2] AS ikey                                                                                                                                                    -- SELECT array[]::text[] --                                                                                                                                                 ) AS c1                                                                                                                                                                      ) AS fatal_query                                                                                                                                                             )                                                                                                                                                                            SELECT * FROM abc WHERE c_text = '"abrakadabra"';
                      QUERY PLAN
------------------------------------------------------
 Result  (cost=0.00..0.00 rows=0 width=36)
   Output: (fatal_query.keys).f2, 'abrakadabra'::text
   One-Time Filter: false
 Query Identifier: 9159242400958923558
(4 rows)
```
